test/api/schemas/token/success.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
You should apply [jsonapi](https://jsonapi.org/)

Reply:
@suho Please ignore this implementation, it's for POW purposes only. 
I will correct it in #25